Configuration/Eras/python/Era_Run2_2017_NewFPix_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+from Configuration.Eras.Modifier_phase1PixelNewFPix_cff import phase1PixelNewFPix
+
+Run2_2017_NewFPix = cms.ModifierChain(Run2_2017, phase1PixelNewFPix)
```

Comment:
My idea is to remove the `Run2_2017_NewFPix` era at the same time the "new geometry" workflows become the default (i.e. replace the "old geometry" workflows). At the same time the customization that is now done with `phase1PixelNewFPix` should be moved to be done with `phase1Pixel`.